src/ui/components/Library/Viewer.tsx

```diff
@@ -47,26 +48,45 @@ function DownloadLinks() {
   );
 }
 
+const filterRecordings = (recordings: Recording[], filters: LibraryFilters) => {
+  const { searchString, qualifiers } = filters;
+  let filteredRecordings = recordings;
+
+  filteredRecordings = searchString
+    ? recordings.filter(
+        r => subStringInString(searchString, r.url) || subStringInString(searchString, r.title)
+      )
+    : recordings;
+  filteredRecordings =
+    qualifiers.target && qualifiers.target === "target:node"
+      ? filteredRecordings.filter(r => !r.user)
+      : filteredRecordings;
+  filteredRecordings =
+    (qualifiers.created && new Date(qualifiers.created))
+      ? filteredRecordings.filter(
+          r => new Date(r.date).getTime() > new Date(qualifiers.created!).getTime()
+        )
+      : filteredRecordings;
+
+  return filteredRecordings;
+};
```

Comment:
this is actually really interesting. I hadn't thought about how we could parse on the client and then just pass variables to our graphQL recordings query...